ci/test.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -e
+set -x
+
+this_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+proj_dir=${this_dir}/../
```

Comment:
we should do the consistent thing of `pwd` for `proj_dir` as well so it is the pwd full path.